backend/fms_core/tests/test_services/individual_test.py

```diff
@@ -36,4 +36,86 @@ def test_get_taxon_invalid(self):
         taxon, errors, warnings = get_taxon(name='Not a valid taxon name')
         self.assertIsNone(taxon)
         self.assertTrue(len(errors) > 0)
-        self.assertTrue('No taxon identified' in errors[0])
\ No newline at end of file
+        self.assertTrue('No taxon identified' in errors[0])
+
+    def test_get_reference_genome(self):
+        TEST_GENOME = "GRCh38.p14"
+
+        reference_genome, _, _ = get_reference_genome(assembly_name=TEST_GENOME)
+        self.assertIsNotNone(reference_genome)
+        self.assertEqual(reference_genome.assembly_name, TEST_GENOME)
+
+    def test_get_or_create_individual_creation(self):
+        HOMO_SAPIENS_NAME = 'Homo sapiens'
+        TEST_GENOME = "GRCh38.p14"
+        
+        taxon, _, _ = get_taxon(name=HOMO_SAPIENS_NAME)
+        reference_genome, _, _ = get_reference_genome(assembly_name=TEST_GENOME)
+        individual, created, errors, warnings = get_or_create_individual(name="BobLeBricoleur",
+                                                                         alias="sj1",
+                                                                         sex="M",
+                                                                         taxon=taxon,
+                                                                         pedigree="BobLeBricoleur",
+                                                                         cohort="BOBS",
+                                                                         reference_genome=reference_genome)
+        self.assertIsNotNone(individual)
+        self.assertTrue(created)
+        self.assertEqual(errors, [])
+        self.assertEqual(warnings, [])
+        self.assertEqual(individual.reference_genome.assembly_name, TEST_GENOME)
+        self.assertEqual(individual.taxon.name, HOMO_SAPIENS_NAME)
+
+    def test_get_or_create_individual_get(self):
+        HOMO_SAPIENS_NAME = 'Homo sapiens'
+        TEST_GENOME = "GRCh38.p14"
+        
+        taxon, _, _ = get_taxon(name=HOMO_SAPIENS_NAME)
+        reference_genome, _, _ = get_reference_genome(assembly_name=TEST_GENOME)
+        _, _, _, _ = get_or_create_individual(name="BobLeBricoleur",
```

Comment:
The names of your variables always make me laugh.

Reply:
Is that good ?

Reply:
Yes hahah